lms/djangoapps/courseware/grades.py

```diff
@@ -543,16 +563,17 @@ def _progress_summary(student, request, course, field_data_cache=None, scores_cl
                     if correct is None and total is None:
                         continue
 
-                    scores.append(
-                        Score(
-                            correct,
-                            total,
-                            graded,
-                            module_descriptor.display_name_with_default,
-                            module_descriptor.location
-                        )
+                    score = Score(
+                        correct,
+                        total,
+                        graded,
+                        module_descriptor.display_name_with_default,
+                        module_descriptor.location
                     )
 
+                    scores.append(score)
+                    locations_to_scores[module_descriptor.location] = score
```

Comment:
These are weighted because that's what's returned by [`get_score()`](https://github.com/mcgachey/edx-platform/blob/lti-collated-grades/lms/djangoapps/courseware/grades.py#L690). Raw scores are what's actually stored in the database tables in MySQL. Weighted scores are what happens after the descriptor's `weight` attribute is applied. We should actually create different types for them and be more disciplined about it, but in the meanwhile, we can name the variables so it's clear which kind it is we're holding.